packages/ember-metal/lib/properties.js

```diff
@@ -136,10 +136,8 @@ export function defineProperty(obj, keyName, desc, data, meta) {
         obj[keyName] = data;
       }
     } else {
-      value = desc;
```

Comment:
You're right. I reverted this line and added a test to `didDefineProperty` hook like the other tests. Thanks!